[
    {
        "url": "https://cat.indores.fr/geonetwork/srv/api/records/611983b9-f635-42f5-bede-3418bea5c8f1/attachments/Falaise%20de%20la%20Mine%20d'Or%20-%202019%20-%20LGO.jpg",
        "size": 1093728,
        "lastModification": 1615454729000,
        "metadataId": 5614,
        "metadataUuid": "611983b9-f635-42f5-bede-3418bea5c8f1",
        "filename": "Falaise de la Mine d'Or - 2019 - LGO.jpg",
        "version": null,
        "metadataResourceExternalManagementProperties": null,
        "approved": true,
        "id": "611983b9-f635-42f5-bede-3418bea5c8f1/attachments/Falaise%20de%20la%20Mine%20d'Or%20-%202019%20-%20LGO.jpg",
        "visibility": "PUBLIC"
    }
]